src/common/services/stockService.js

```diff
@@ -3,7 +3,8 @@
 
 	function StockService ($http, ApiUrl, $q)  {
 		function getStocks () {
-			//fake data call to a fake database
+			// TODO set up calls to 
```

Comment:
let's put this todo comment all on one line, that way when we run `gulp todo` it will print a list of them